packages/aws-cdk/lib/notices.ts

```diff
@@ -485,13 +488,18 @@ export interface NoticeDataSource {
 export class WebsiteNoticeDataSource implements NoticeDataSource {
   private readonly options: SdkHttpOptions;
 
-  constructor(private readonly ioMessages: IoDefaultMessages, options: SdkHttpOptions = {}) {
+  constructor(private readonly ioHelper: IoHelper, options: SdkHttpOptions = {}) {
```

Comment:
Looks like you didn't really need to change this? You didn't use `this.ioHelper` anywhere new and the message emitted is the same?